vaccine_feed_ingest/utils/normalize.py

```diff
@@ -242,11 +242,20 @@ def normalize_phone(phone: Optional[str]) -> Optional[List[str]]:
     phone = phone.replace(" option ", " ext. ")
     phone = phone.replace(" press ", " ext. ")
 
+    # The library doesn't match 311 etc; use a placeholder and then revert.
+    phone = re.sub(r"\b211\b", "(900) 211-0000", phone)
+    phone = re.sub(r"\b311\b", "(900) 311-0000", phone)
+
     phones = []
     for match in phonenumbers.PhoneNumberMatcher(phone, "US"):
-        phones.append(
-            phonenumbers.format_number(
-                match.number, phonenumbers.PhoneNumberFormat.NATIONAL
+        if match.number.national_number == 9002110000:
+            phones.append("211")
```

Comment:
@bryanculbertson - do we expect downstream systems to be able to handle these?

Reply:
@cdjj I don't think N11 codes will work with down systems. Can you add either skip these phone codes, or add these codes to the `other` field instead of to the `phone` field?

Reply:
Hmm, we'll need to change this so it returns Contact objects rather than a list of strings that then get turned into Contacts by the normalizers, but that's straightforward.

Reply:
OK, refactored; PTAL.